src/interface/src/styles/material.scss

```diff
@@ -23,3 +23,12 @@
 .mat-table .mat-header-cell {
   font-weight: 700;
 }
+
+// default styles for menu
+body {
+  --mat-menu-item-label-text-font: Public Sans;
```

Comment:
nitty question: Does it matter if we put the font name in quotes in scss? Should we have a convention?

Reply:
good catch, adding quotes!